packages/vm/.eslintrc.js

```diff
@@ -1,18 +1,10 @@
 module.exports = {
   extends: "@ethereumjs/eslint-config-defaults",
-  ignorePatterns: ["tests", "scripts", "examples", "karma.conf.js"],
+  ignorePatterns: ["scripts", "examples", "karma.conf.js"],
   rules: {
-    '@typescript-eslint/await-thenable': 'off',
-    '@typescript-eslint/no-floating-promises': 'off',
-    '@typescript-eslint/no-unused-vars': 'off',
     '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/restrict-plus-operands': 'off',
-    'no-async-promise-executor': 'off',
     'no-invalid-this': 'off',
```

Comment:
ESLint is confused since we use the variable name `this` in things like `runBlock`. We can enable this, but TypeScript will fail to compile if you violate this rule.